src/components/CategoryCheckboxes/Component.ts

```diff
@@ -0,0 +1,122 @@
+/* tslint:disable */
```

Comment:
what changed in this file?feels like a new code

Reply:
File name changed, it flagged it as new code